recipes/ocean_wave_tracing/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "ocean-wave-tracing" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ocean_wave_tracing-{{ version }}.tar.gz
+  sha256: 54a06d09a0b57683686405ba15dbf51aed03a85ebd05a20321e2060bb1fae073
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Why do you need a c compiler for this package?

Reply:
Thanks for pointing out. I don't. Removed